web/react/components/access_history_modal.jsx

```diff
@@ -3,7 +3,8 @@
 
 var UserStore = require('../stores/user_store.jsx');
 var AsyncClient = require('../utils/async_client.jsx');
-var Utils = require('../utils/utils.jsx');
+var LoadingScreen = require('./loading_screen.jsx');
+var utils = require('../utils/utils.jsx');
```

Comment:
Because it's lowercase everywhere else in the code (I'm sure that fact might come up when the style guide goes in) so I changed it to match.